msteams-platform/concepts/build-and-test/apps-localization.md

```diff
@@ -157,4 +157,4 @@ If you provide localized versions of your application, the users respond with th
 
 ## See also
 
-[Localize JSON schema reference](~/resources/schema/localization-schema.md)
+* [Localize JSON schema reference](~/resources/schema/localization-schema.md)
```

Comment:
@ashishguptaiitb 

See also has only link and we won't add bullet for one item.